monitoring/poison_message_queue_finder.py

```diff
@@ -141,5 +141,25 @@ def get_connection_stats():
             print(json.dumps(json_to_dump))
 
 
+def get_bad_message_counts():
+    response = requests.get(f'{Config.EXCEPTIONMANAGER_URL}/badmessages/summary?minimumSeenCount={Config.BAD_MESSAGE_MINIMUM_SEEN_COUNT}')
+    response.raise_for_status()
+
+    messages = response.json()
+    queue_counts = {}
+    for message in messages:
+        if message['affectedQueues'][0] not in queue_counts:
```

Comment:
This will only log the counts for the first queue in the list of any particular bad message. We probably want to count all of them for each? I can see it either way, if we count them all we'll get multiple alerts off the back of one bad message if it touches multiple queues.